snuba/clickhouse/native.py

```diff
@@ -269,9 +299,14 @@ def __transform_result(self, result: Sequence[Any], with_totals: bool) -> Result
         if with_totals:
             assert len(data) > 0
             totals = data.pop(-1)
-            new_result = {"data": data, "meta": meta, "totals": totals}
+            new_result = {
+                "data": data,
+                "meta": meta,
+                "totals": totals,
+                "profile": profile,
+            }
         else:
-            new_result = {"data": data, "meta": meta}
+            new_result = {"data": data, "meta": meta, "profile": profile}
```

Comment:
Does that mean our API is returning "profile" information now?

Reply:
Yes, the data is there, just not being used yet. I am going to make use of it in separate PR(s).